src/docs/BUILD.md

```diff
@@ -25,63 +48,89 @@ The path to the openvino install directory is referred as <INSTALL_DIR> througho
     source <INSTALL_DIR>/setupvars.sh
     cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
     cmake --build ./build/ --config Release --target package -j
-    cmake --install ./build/ --config Release --prefix ov
+    cmake --install ./build/ --config Release --prefix <INSTALL_DIR>
     ```
+    > **NOTE**: For running setupvars script on Windows, use command `call <INSTALL_DIR>\setupvars.bat`
 
-## Build for Windows Systems
-
-### Software Requirements 
+To optimize the package size, you can reduce the ICU (International Components for Unicode) data size when building OpenVINO Tokenizers.
+For more information please refer to the [OpenVINO Tokenizers instructions](https://github.com/openvinotoolkit/openvino_tokenizers?tab=readme-ov-file#reducing-the-icu-data-size).
 
-- [CMake](https://cmake.org/download/) 3.23 or higher
-- Microsoft Visual Studio 2019 or higher, version 16.3 or later
-- Python 3.8 or higher
-- Git for Windows
+### Build OpenVINO GenAI only
```

Comment:
OpenVINO must always be built if GenAI is built because of ABI issue you described above. The idea of this section was to allow omitting `source <INSTALL_DIR>/setupvars.sh` and `cmake --install ./build/ --config Release --prefix <INSTALL_DIR>`
You did a great job describing environment set up. Replace `source <INSTALL_DIR>/setupvars.sh` from `Build OpenVINO and OpenVINO GenAI from sources` section with that description. Please add PATH env var replacing LD_LIBRARY_PATH for Wndows case.
> 2. Set up the environment:
>  - Option 1 - using OpenVINO `setupvars.sh` script:
      source <INSTALL_DIR>/setupvars.sh
>  - Option 2 - setting environment variable manually:
      export OpenVINO_DIR=<INSTALL_DIR>/runtime
      export PYTHONPATH=<INSTALL_DIR>/python:./build/:${PYTHONPATH}
      export LD_LIBRARY_PATH=<INSTALL_DIR>/runtime/lib/intel64:${LD_LIBRARY_PATH}
      

Do the same for `cmake --install ./build/ --config Release --prefix <INSTALL_DIR>` step in `Build OpenVINO and OpenVINO GenAI from sources` section. Explain that it's needed to keep artefacts together and the alternative solution is to set env vars:
```sh
export PYTHONPATH=<GENAIA_ROOT_DIR>/build/:${PYTHONPATH}
export LD_LIBRARY_PATH=<GENAIA_ROOT_DIR>/build/openvino_genai/:${LD_LIBRARY_PATH}
```
Add PATH for windows as well.

Doing so allows to remove `Build OpenVINO GenAI only` section completely.

Reply:
Extended "Build everything from source" instructions, removed "Build OpenVINO GenAI only" section